src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1104,10 +1104,11 @@ void EIRPlantLoopHeatPump::checkConcurrentOperation(EnergyPlusData &state)
         }
     }
 }
+
 void EIRPlantLoopHeatPump::oneTimeInit(EnergyPlusData &state)
 {
     // This function does all the one-time initialization
-    std::string static const routineName = std::string("EIRPlantLoopHeatPump :") + __FUNCTION__;
+    std::string static const routineName = std::string("EIRPlantLoopHeatPump : oneTimeInit"); // + __FUNCTION__;
```

Comment:
Does this need to be a `const std::string` can it not be a `constexpr std::string_view`?

Reply:
Addressed in earlier commits. Using ` constexpr std::string_view routineName` now; also applied the same changes to function in the parent class.